### PR TITLE
Adding redirect rules 

### PR DESCRIPTION
**Pull request by flossypurse**

### CODE REVIEW

**Review by sw-yx — APPROVED** [review]

File: netlify.toml [hunk 1]
```diff
@@ -4,3 +4,7 @@
     [plugins.inputs]
 
     skipPatterns = ["#!", "www.google-analytics.com", "www.googletagmanager.com"]
+
+[[redirects]]
+  from = "/docs/overview"
+  to = "/docs/concept-overview"
```
Comment (sw-yx) [inline netlify.toml:10]: i think not necessary if you have `_redirects` in the right place

Comment (flossypurse) [reply netlify.toml:10]: The _redirects file needs to be in the build folder - 
With Docusaurus the build folder is created when the site is built.
So, alternative is to use the .toml file - which we already have and doesn't need to be any place other than the root directory.

File: _redirects [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/docs/overview   /docs/concept-overview
+/docs/activities /docs/concept-activities
+/docs/workflows  /docs/concept-workflows
```
Comment (sw-yx) [inline _redirects:3]: i think this file needs to be in `static`, aka `/static/_redirects`

File: netlify.toml (hunk 1 above)
Comment (sw-yx) [reply netlify.toml:10]: we can also put it in `/static` :)

Comment (flossypurse) [reply netlify.toml:10]: probably, but the .toml file is already there and provides more comprehensive redirect options -
And it works -

**Pull request merged**